client/js/app.js

```diff
@@ -59,6 +61,10 @@ const init = () => {
     makeSticky(stickyEls, store$);
   }
 
+  if (galleryCaptions) {
+    truncateText(galleryCaptions);
```

Comment:
Why not have the `truncateText` only aware of 1 element?
e.g. `document.querySelectorAll('.js-image-gallery .captioned-image__caption-text').forEach(truncateText);`?